### ISSUE TITLE
Have the google translate prompt pop up on a few key pages

### ISSUE THREAD

**Issue by DheekshuKumar** [CovEducation/CovEd#59]
More to be discussed re: what pages should have this prompt. All? Some? -- but if anyone has thoughts on this, run it by Dheekshu, and we can discuss implementation.

**Comment by oupadhyay** [comment]
## Google Translate API Pricing
Looking at the pricing, I think this is pretty expensive given how many people will be on the website.
![Pricing](https://user-images.githubusercontent.com/46334571/80893909-4db0da00-8c8b-11ea-88eb-20da41dcd0b7.png)
## Another solution
Preload translations and have a menu for the languages (I am thinking English, French, Mandarin, Spanish, and any others). Then, based the menu selection for language, have the site update or maybe just the text. 
### The text update
This would require some sort of class for all the text fields that need updating.
### Site Update
Basically it would be a refresh lol.

**Comment by johancc** [comment]
"When you visit a page in Chrome where the language doesn't match your locale, Chrome will prompt you to translate the page content"
If people set their preferred language to be something other than English, our site would be translated on the client side

**Comment by oupadhyay** [comment]
I think this is resolved. The client side deals with translation.
